doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3781,7 +3857,7 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
```

Comment:
.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-22443: A rare assert when disabling a periodic advertising set with responses
  In some rare cases, the controller can assert when disabling a periodic advertising set with responses

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
DRGN-22441: Fixed an issue where the length byte of the HCI packet could be incorrect
  This could happen when the packet contained an LE BIG Sync Established event or LE BIG Complete event with status not equal to success.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-22652:  Assert when receiving on the S8 Coded PHY
  This could happen in a connection where link peer device is transmitting on  on S8 Coded PHY.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
DRGN-22686: Missing truncated advertising report
  The extended scanner would not generate a truncated advertising report after the coexistence interface aborted the reception of an ``AUX_CHAIN_IND`` packet.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
DRGN-22678: A very rare issue where the controller stopped generating advertising reports
  On nRF52 Series and nRF53 Series devices, this would happen at least one hour after the scanner started.
  On nRF54L and nRF54H Series devices, this would occur immediately after the scanner started. 
  It would only happen when one of the following applies:
    * There was another central-like scheduling activity running. Examples of roles with such activities are the ACL central, periodic advertiser, isochronous broadcaster and the CIS central.
      This activity was configured with an event length or event spacing equal or greater than the scan interval.
      This is typically only true for use cases where the application enables isochronous channels or uses very short scan windows.
    * The scanner was configured with scan window equal to scan interval (continuous scanning).
    * The central-like scheduling activity required less than 1 ms to complete at the point in time where the scanner started.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 
DRGN-22230: A rare issue where the scanner would be stuck in the synchronizing state after failing to receive an ``AUX_ADV_IND`` packet
  This could only happen when the corresponding ``ADV_EXT_IND`` packet contained a resolvable address, private address resolution is enabled, and the periodic advertising list is not used. 

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-22705: The controller could generate the LE Advertising Set Terminated event one event sooner than expected
  This could only happen when all of the following apply:
    * a non-zero Max_Extended_Advertising_Events parameter was used in the LE Set Extended Advertising Enable command.
    * other ongoing activities in the controller prevented the first advertising event from taking place when the advertising set was created.

22930

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
DRGN-22879: The Central could fail to receive the last packet in an isochronous event
  This could only happen if the Connected Isochronous Stream Creation procedure was initiated by the host before the Encryption Start procedure completed.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 
DRGN-23002: An assert could happen when using the coexistence interface
  This could happen when any of the following controller activities were ongoing:
    * Isochronous Broadcaster
    * Connected Isochronous channel in the peripheral role
    * Periodic Sync with Responses

.. rst-class:: v2-7-0
DRGN-23291:  LE Power Control was not being used for CIS
  This could happen when the CIS was not the first CIS in the CIG.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-23204: The SoftDevice Controller in the peripheral role could terminate a connection due to a MIC failure during a valid encryption start procedure
  This could only happen if the ``LL_ENC_RSP`` packet was corrupted due to on-air interference.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-23586: The received unframed Isochronous SDUs were not reported to be a SDU interval apart
  This could happen when the ISO interval is greater than the SDU interval and multiple SDUs can be received in a single ISO interval.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
DRGN-23693: Wrong sleep clock accuracy
The sleep clock accuracy communicated to the peer was too inaccurate if MPSL was initialized with a low frequency clock accuracy better than 20ppm

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
DRGN-22036: A rare issue in the controller that could lead to a bus fault
  This could only happen when all of the following conditions were met:
    * The host was too slow at pulling HCI events.
    * One or more HCI events had been masked in the controller.
    * The controller was raising ACL or ISO data to the host

Reply:
Added. And these are all fixed in 2.8.0, right?

Reply:
Sorry that is a question for Dragoon, I'm just dealing with Octave issues.

Reply:
Asking @laxiLang as he wanted to add these issues.